### ISSUE TITLE
"device_state_attributes" deprecated in Home Assistant 2021.12.5

### ISSUE THREAD

**Issue by mbauwens** [barleybobs/homeassistant-ecowater-softener#8]
While running the custom component in Home Assistant 2021.12.5, I get the error:

```
Logger: homeassistant.helpers.entity
Source: helpers/entity.py:549
First occurred: 04:38:55 (1 occurrences)
Last logged: 04:38:55

Entity sensor.ecowater (<class 'custom_components.ecowater_softener.sensor.EcowaterSensor'>) implements device_state_attributes. Please report it to the custom component author.
```

Can't find the exact release notes, but it seems that (since HA 2021.11.0) attributes are moving towards separate sensors - I think that's what this relates to?

**Comment by ryphez** [comment]
+1 I have the same issue on 2021.12.9

**Comment by mark007** [comment]
+1, also seeing this one. Is this integration still active, or should someone look to contribute a fix?

**Comment by mbauwens** [comment]
I think @barleybobs might be somewhat on a break or something because I also received no response to my PR from December for a different issue ( #6 #7).

**Comment by barleybobs** [comment]
Updating it now! Sorry I haven't been very active lately.

Thanks 👍

**Comment by barleybobs** [comment]
Fixed in #9

**Comment by barleybobs** [comment]
Fixed in #9
